src/test/java/de/retest/recheck/cli/util/ReportCreator.java

```diff
@@ -1,21 +1,98 @@
 package de.retest.recheck.cli.util;
 
 import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.rules.TemporaryFolder;
 
 import de.retest.recheck.LoadRecheckIgnoreUtil;
 import de.retest.recheck.ReplayResultProvider;
 import de.retest.recheck.persistence.RecheckReplayResultUtil;
+import de.retest.recheck.report.ActionReplayResult;
 import de.retest.recheck.report.SuiteReplayResult;
+import de.retest.recheck.report.TestReplayResult;
+import de.retest.recheck.report.action.ActionReplayData;
+import de.retest.recheck.report.action.DifferenceRetriever;
+import de.retest.recheck.report.action.WindowRetriever;
+import de.retest.recheck.ui.DefaultValueFinder;
+import de.retest.recheck.ui.Path;
+import de.retest.recheck.ui.descriptors.Attribute;
+import de.retest.recheck.ui.descriptors.Attributes;
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.descriptors.PathAttribute;
+import de.retest.recheck.ui.descriptors.RootElement;
+import de.retest.recheck.ui.descriptors.StringAttribute;
+import de.retest.recheck.ui.descriptors.SutState;
+import de.retest.recheck.ui.descriptors.TextAttribute;
+import de.retest.recheck.ui.diff.RootElementDifference;
+import de.retest.recheck.ui.diff.RootElementDifferenceFinder;
 
 public class ReportCreator {
```

Comment:
We should consider moving this helper to recheck and generalize all path related stuff.